spec/support/transactions/context.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2019 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  module Transactions
```

Comment:
Why not just define a `transform_arguments` method on `CRUD::Operation` and then override it as needed (e.g. in the transactions tests) instead of defining an entire new module, and then passing it into every method for a CRUD operation?

Reply:
Passing of context in all methods is temporary. Previously the operation class mixed operation definitions, operation execution, test runner state and outcome expectations. This PR separates runner state out into the context classes. The remaining three parts remain in the operation classes for the time being, but long term will be better organized and will not have the current repetition of the context parameter.

Reply:
Okay, I'm fine with fixing this in a later commit